cmd/corald/service/service.go

```diff
@@ -2,54 +2,17 @@ package service
 
 import (
 	"net/http"
-	"time"
 
-	"github.com/ardanlabs/kit/log"
 	"github.com/ardanlabs/kit/web/app"
 	"github.com/coralproject/shelf/internal/platform/auth"
-	"github.com/pborman/uuid"
 )
 
-// SignServiceRequest will take a given signer, and adds a Authorization header
-// with the token that is generated from the signer.
+// SignServiceRequest signts a request with the claims necisary to authenticate
+// with downstream services.
 func SignServiceRequest(context interface{}, signer auth.Signer, r *http.Request) error {
```

Comment:
Is there a need for this function at this point?  Why not wrap this into the `auth` package at the top level API or just call `SignRequest`.